changelog.md

```diff
@@ -17,6 +17,7 @@ which is particularly useful in very large workflows.
 
 ### New components
 - Added component `fast_ani`.
+- Added component `bcalm`.
```

Comment:
Hmm this should be on the next release of the dev branch and not under the `1.3.1` release. But to do so you will need to merge the dev branch into your fork so that you can you can resolve the conflicts with `dev`